source/tests/system/nidaqmx_driver_api_tests.cpp

```diff
@@ -712,6 +737,40 @@ TEST_F(NiDAQmxDriverApiTests, AIVoltageChannel_ReadBinaryI16_Succeeds)
   EXPECT_EQ(NUM_SAMPS, response.samps_per_chan_read());
 }
 
+TEST_F(NiDAQmxDriverApiTests, AIVoltageChannel_ReadRaw_Succeeds)
+{
+  using TRaw = uint16_t;
+  create_ai_voltage_chan(-5.0, 5.0);
+
+  start_task();
+  ReadRawResponse response;
+  const auto NUM_SAMPS = 10;
+  auto status = read_raw<TRaw>(NUM_SAMPS, response);
+  stop_task();
+
+  EXPECT_SUCCESS(status, response);
+  EXPECT_EQ(NUM_SAMPS, response.samps_read());
+  EXPECT_EQ(NUM_SAMPS * sizeof(TRaw), response.read_array().size());
+  auto data_ptr = reinterpret_cast<const TRaw*>(response.read_array().data());
+  auto data_vector = std::vector<TRaw>(data_ptr, data_ptr + NUM_SAMPS);
+  EXPECT_THAT(data_vector, Each(Not(Eq(0))));
```

Comment:
I might be missing something but is it guaranteed that each data point read isn't 0 or is it just extremely unlikely?